src/main/scala/gestalt/dotty/Expander.scala

```diff
@@ -67,7 +76,8 @@ object Expander {
   /** Expand def macros */
   def expandDefMacro(tree: tpd.Tree)(implicit ctx: Context): untpd.Tree = tree match {
     case ExtractApply(Select(obj, method), targs, argss) =>
-      val className = obj.symbol.info.classSymbol.fullName + "$inline$"
+      val objType = obj.symbol.info.resultType
+      val className = javaClassName(objType.classSymbol) + "$inline$"
```

Comment:
This issue appears when `object.info` returns a `MethodType`, then these two act differently. For example, calling the constructor `new plus2(3)` or when it is applying a method with arguments `Generator.fromSeed(20).apply(3)`.
If we look at `dotty.tools.dotc.core.Types.Type#classSymbol` it does not have a case for `MethodType` so it returns the `NoSymbol` which does not get us the correct class.